hexrd/ui/image_series_toolbar.py

```diff
@@ -75,7 +75,8 @@ def update_name(self, name):
             self.name = name
 
     def set_visible(self, b=False):
-        self.widget.setVisible(b)
+        size = len(self.ims) - 1 if self.ims else 0
```

Comment:
Is this test checking if we have images? If so I think we could do `self.widget.setVisible(b and bool(self.ims))`, may be I am miss understand what this check is doing ...

Reply:
I think it's checking to see if we have more than one image in the imageseries.

Reply:
The check is making sure we have more than one frame to view. Currently even `.tiff` images are loaded as an ImageSeries so that they can be rotated, etc as needed.

Reply:
It might be a little clearer if it is something like `self.widget.setVisible(b and self.ims and len(self.ims) > 1)`, or if `self.ims` is guaranteed to be a dict, `self.widget.setVisible(b and len(self.ims) > 1)`.

Reply:
Since everything is now being loaded through a shared pathway `self.ims` should always be a dict, so I've updated the check accordingly:
`self.widget.setVisible(b and len(self.ims) > 1)`.

Reply:
That looks better, thanks